### ISSUE TITLE
Disable autofocus on mobile devices

### ISSUE THREAD

**Issue by rajbirjohar** [pacocoursey/cmdk#127]
When the menu is open on mobile devices, the `<Command.Input />` element grabs focus automatically which is expected.

Now the issue is that the mobile keyboard also pops up and covers most of the items. Is there a way to disable the auto focus on mobile devices? I've already tried a number of possible solutions such as `preventDefault()`, setting `autoFocus={false}`, setting `tabIndex="-1"`, creating a second "hidden" input (which also grabbed focus) and nothing worked. It's kind of a pain to open the menu and then have to collapse the keyboard every time I want to scroll down.

Am I missing something or is it explained in the documentation somewhere?

**Comment by pacocoursey** [comment]
Yeah this is annoying, what you want is to change the `onOpenAutoFocus` event from Radix's Dialog.Content component, but we currently don't support passing props through to that element.

**Comment by rajbirjohar** [comment]
That makes sense @pacocoursey. Would we be expecting this to come out in a future release? Or would it be a difficult feature to implement? I can try taking a look.

**Comment by menachem1** [comment]
is there an update or workaround for this?

**Comment by menachem1** [comment]
#213 https://github.com/pacocoursey/cmdk/pull/213

**Comment by ImJustChew** [comment]
Are there any plans to fix this? and are there any temporary workarounds for this?

**Comment by Jayprecode** [comment]
I'd suggest you can set the font-size of the input to 16px for mobile device specifically to prevent autofocus, especially on ios..

This might solve your problem, just a suggestion tho

**Comment by V-iktor** [comment]
This autofocus makes the input unusable on mobile

**Comment by ritiksharmarj** [comment]
This solve my problem
```js
 <PopoverContent
      ref={ref}
      asChild // accessible with keyboard navigation
      className={cn(
        "w-auto min-w-[var(--radix-popover-trigger-width)] p-0",
        className,
      )}
      onOpenAutoFocus={(e) => !isDesktop && e.preventDefault()} // disable autofocus on mobile devices
      {...props}
    >
      <Command loop className="w-full">
        {children}
      </Command>
    </PopoverContent>
```
